4.md

```diff
@@ -0,0 +1,4 @@
+Как зарегистрированный пользователь, 
```

Comment:
There are too many things in this story. Make more smaller stories, please.

Reply:
Splitted into two stories: 6, 7